test/ClangImporter/newtype_conformance.swift

```diff
@@ -16,12 +16,11 @@ func acceptHashable<T: Hashable>(_: T) {}
 // expected-note@-1 {{where 'T' = 'WrappedRef'}}
 // expected-note@-2 {{where 'T' = 'WrappedValue'}}
 func acceptComparable<T: Comparable>(_: T) {}
-// expected-note@-1 {{where 'T' = 'NSNotification.Name'}}
 
 func testNewTypeWrapper(x: NSNotification.Name, y: NSNotification.Name) {
   acceptEquatable(x)
   acceptHashable(x)
-  acceptComparable(x) // expected-error {{global function 'acceptComparable' requires that 'NSNotification.Name' conform to 'Comparable'}}
+  acceptComparable(x) // expected-error {{type 'NSNotification.Name' does not conform to protocol 'Comparable'}} {{19-19=.rawValue}}
```

Comment:
Hmm, is it possible to keep the `global function requires that` part, or at least a note pointing to the requirement? 

On second thought, I think it would be nice if the original error stays the same, and add a note saying "did you mean .rawValue" or similar with the fix-it. What do you think?

Reply:
Yeah, I think that would make sense, I'd still keep `UseRawValue` fix, to dial with ambiguities, but would diagnose `MissingConformanceFailure` with a tailored note.

Reply:
✅

Reply:
What do you think?